### ISSUE TITLE
expose "modified" attribute

### ISSUE THREAD

**Issue by seansay** [siznax/wptools#27]
I saw the api reponse has this value to indicate the page's last modified, can we add this to `wptools`?

**Comment by seansay** [comment]
seems only `get_rest` has this attr set,

but wikidata query also return `https://www.wikidata.org/w/api.php?action=wbgetentities&formatversion=2&ids=Q5094115&languages=zh-cn&props=info%7Cclaims%7Cdescriptions%7Clabels%7Csitelinks&redirects=yes&sites=&titles=&variant=zh-cn`

**Comment by siznax** [comment]
Thanks for your request @seansay. I pushed a fix for this 4101cb473cc74e77b386f9d457fb2701d7523ef1:

```python
>>> p = wptools.page(wikibase='Q5094115', lang='zh', variant='zh-cn').get_wikidata()
www.wikidata.org (action=wikidata) Q5094115
www.wikidata.org (action=wikidata) Q148|Q5
林一峰 (zh/zh-cn)
{
  lang: zh
  modified: 2016-10-25T03:41:57Z
  title: 林一峰
  variant: zh-cn
  wikibase: Q5094115
  ...
}
```

**Comment by siznax** [comment]
It turns out that the modified attribute is ambiguous. If you invoke `get()` then the modified date of the page is clobbered by the modified date of the Wikidata item. We should make modified a dict or list.
